src/pudl/etl/__init__.py

```diff
@@ -86,6 +86,9 @@ def load_dataset_settings_from_file(setting_filename: str) -> dict:
     ).datasets.dict()
 
 
+# Set env vars from .pudl.yml if not already set
+pudl.workspace.setup.get_defaults()
```

Comment:
I would love if we could avoid loading the settings at import time...

Reply:
We could also say, 'if you don't set the env vars you don't get to run ETLs from dagit.'

Reply:
Agreed in Slack to just not try to set env vars automatically here.